sdcm/utils/aws_region.py

```diff
@@ -31,13 +31,14 @@ class AwsRegion:
     SCT_VPC_CIDR_TMPL = "10.{}.0.0/16"
     SCT_SECURITY_GROUP_NAME = "SCT-2-sg"
     SCT_TEST_SECURITY_GROUP_NAME_TMPL = "SCT-2-sg-{}"
-    SCT_SUBNET_NAME = "SCT-2-subnet-{availability_zone}"
+    SCT_SUBNET_NAME = "SCT-2-subnet-{availability_zone}{subnet_index}"
     SCT_INTERNET_GATEWAY_NAME = "SCT-2-igw"
-    SCT_ROUTE_TABLE_NAME = "SCT-2-rt"
+    SCT_ROUTE_TABLE_NAME = "SCT-2-rt{index}"
     SCT_KEY_PAIR_NAME = "scylla-qa-ec2"  # TODO: change legacy name to sct-keypair-aws
     SCT_SSH_GROUP_NAME = 'SCT-ssh-sg'
 
-    def __init__(self, region_name):
+    def __init__(self, region_name, subnets_per_az=1):
```

Comment:
this should be subnets_per_az=2 by default
or event this should be an input to this class, maybe defined on the class like SCT_* values above

Reply:
done